neo/Network/RPC/Helper.cs

```diff
@@ -0,0 +1,108 @@
+using Neo.Cryptography.ECC;
+using Neo.SmartContract;
+using Neo.Wallets;
+using System;
+using System.Numerics;
+
+namespace Neo.Network.RPC
+{
+    public static class Helper
```

Comment:
Looks good to me.
Let's discuss and decide what will be moved to Plugin, perhaps, also this?

Reply:
This is a Helper class of this namespace. How can this be moved to Plugin?

Reply:
Do we really need this class?

Reply:
This class will be very convenient to deal with different formats of string, we can provide more kinds of input format when users calling the methods.

Reply:
I think that we can leave it like this and latter move everything to plugin.

But the organization looks good to me.